oat-sa/tao-core-ui-fe
# Feature/CON-30/support responsive video

Related to: https://oat-sa.atlassian.net/browse/CON-30

https://github.com/oat-sa/tao-core/pull/2780
https://github.com/oat-sa/extension-tao-mediamanager/pull/248
https://github.com/oat-sa/extension-tao-test-preview-ui-loader/pull/17
https://github.com/oat-sa/extension-tao-itemqti/pull/1610
https://github.com/oat-sa/tao-item-runner-qti-fe/pull/140

Support video responsive width in Item Authoring.

How to test:

- go to Items Authoring
- add video by Media Interaction and by media object
- try to scale
- check scale in preview
